lib/recurly/client.rb

```diff
@@ -10,18 +10,6 @@ class Client
 
     BASE_URL = "https://partner-api.recurly.com/"
 
-    # The last result of the *X-RateLimit-Limit* header
-    # @return [Integer] The rate limit applied to this client.
-    attr_reader :rate_limit
-
-    # The last result of the *X-RateLimit-Remaining* header
-    # @return [Integer] The number of remaining requests, decrements per request.
-    attr_reader :rate_limit_remaining
-
-    # The last result of the *X-RateLimit-Reset* header
-    # @return [DateTime] The DateTime in which the request count will be reset.
-    attr_reader :rate_limit_reset
```

Comment:
These are technically breaking changes. I want encourage people to use the per-request fields.